### PR TITLE
Translated some src/java programs

### PR DESCRIPTION
**Pull request by BenEklundCS**
Translated:

BuscaBinaria --> BinarySearch
Exponenciacao --> Exponentiation
ExponenciacaoRecursiva --> RecursiveExponentiation
TorreDeHanoi --> TowersOfHanoi

### CODE REVIEW

**Comment by BenEklundCS** [comment]
https://github.com/kelvins/Algoritmos-e-Estruturas-de-Dados/issues/182 Helping here

**Comment by BenEklundCS** [comment]
I'm not 100% sure if the old files are supposed to be renamed, or left as is? 

Looking for help on my next steps. Thanks! :)

File: src/java/BinarySearch.class
Comment (kelvins) [inline src/java/BinarySearch.class:1]: Could you please remove this `.class` file?

**Review by kelvins — COMMENTED** [review]
Thanks @BenEklundCS 🎉 . Yes, we can rename/translate all files to english. I would suggest opening a PR only for renaming them because it would be easier to review.

Could you please fix the CI issues? Looks like the java linter is broken as well as the link checker.

File: src/java/BinarySearch.class
Comment (BenEklundCS) [reply src/java/BinarySearch.class:1]: In trying to edit my PR, I may have Closed it accidentally. I was going to remove my commits. How did this end up causing a force push?

**Pull request closed without merging**